js-constants.js

```diff
@@ -0,0 +1,155 @@
+    icon: "/img/removal/remove-status-in-progress.svg",
+    locale_var: "remove-filter-in-progress",
+  },
+  COMPLETE: {
+    id: "COMPLETE",
+    icon: "/img/removal/remove-status-complete.svg",
+    locale_var: "remove-filter-complete",
+  },
+  BLOCKED: {
+    id: "BLOCKED",
+    icon: "/img/removal/remove-status-blocked.svg",
+    locale_var: "remove-filter-blocked",
+  },
+};
+
+const JS_CONSTANTS = {
+  REMOVE_CHECK_WAITLIST_ENABLED: true, //enable the waitlist check
+  REMOVE_CHECK_EMAIL_DOMAIN_ENABLED: true, //enable a check of the user's email domain to be within the REMOVE_EMAIL_DOMAIN_LIST array
+  REMOVE_WILLINGNESS_TO_PAY_ENABLED: false, //show the willingness to pay screen
+  REMOVE_CHECK_ENROLLMENT_ENDED_ENABLED: false, //allows us to enforce a fixed amount of time from the pilot start (set with REMOVAL_PILOT_ENROLLMENT_END_DAY) for users to enroll when true
+  REMOVE_EMAIL_DOMAIN_LIST: [
+    "mozilla.com",
+    "getpocket.com",
+    "mozillafoundation.org",
+  ], //users must sign up with an email from one of these domains if REMOVE_CHECK_EMAIL_DOMAIN_ENABLED is true
+  KANARY_PRIVACY_LINK: "https://www.thekanary.com/privacy_and_security",
+  REMOVE_CANCELATION_SURVEY_LINK: "", //MH TODO: Get survey link
+  REMOVE_LOGGED_IN_DEFAULT_ROUTE: "/user/remove-data",
+  REMOVE_ACTIVE_LINKS: [
+    "/remove-data",
+    "/remove-data?show_form=true",
+    "/remove-enrolled",
+    "/remove-enroll",
+    "/remove-enroll-full",
+    "/remove-enroll-ended",
+    "/remove-signup-confirmation",
+    "/remove-update-confirmation",
+    "/remove-delete-confirmation",
+    "/remove-pilot-ended",
+  ], //used to highlight the exposures tab on any route in array
+  REMOVE_ROUTES: ["/user/remove-data", "/user/remove-enroll"],
+  REMOVAL_SITES: [
+    "anywho.com",
+    "backgroundalert.com",
+    "beenverified.com",
+    "checkpeople.com",
+    "checkthem.com",
+    "clustrmaps.com",
+    "cocofinder.com",
+    "cyberbackgroundchecks.com",
+    "dobsearch.com",
+    "emailsherlock.com",
+    "familytreenow.com",
+    "fastpeoplesearch.com",
+    "freepeopledirectory.com",
+    "homemetry.com",
+    "hpccusa.com",
+    "infotracer.com",
+    "instantcheckmate.com",
+    "intelius.com",
+    "kiwisearches.com",
+    "locatepeople.com",
+    "mylife.com",
+    "neighborwho.com",
+    "nuwber.com",
+    "peekyou.com",
+    "peoplebyname.com",
+    "peoplefinders.com",
+    "pplcheck.com",
+    "publicdatadigger.com",
+    "publicdatausa.com",
+    "publicemailrecords.com",
+    "publicinfoservices.com",
+    "publicrecordsnow.com",
+    "radaris.com",
+    "searchpeoplefree.com ",
+    "searchquarry.com ",
+    "smartbackgroundchecks.com",
+    "socialcatfish.com",
+    "spokeo.com",
+    "spydialer.com",
+    "spyfly.com",
+    "thatsthem.com",
+    "truthfinder.com",
+    "unmask.com",
+    "usphonebook.com",
+    "ussearch.com",
+    "veripages.com",
+    "whitepages.com",
+    "xlek.com",
+    "yellowbook.com",
+    "zabasearch.com",
+  ],
```

Comment:
Is this info not provided by Kanary?  Not sure we want to be the stewards for the list of data brokers.

Reply:
It's not provided via the API, but it is listed on their site. It's a good callout. I'll ask Kanary if there is a better way to either retrieve this info as an up-to-date list or deep link to the info on their site instead.